### DISCUSSION TITLE
adding titles to plots

### DISCUSSION THREAD

**Discussion by zanderteller** [maartenbreddels/ipyvolume#410] [Q&A] [answered]
I’ve been using IPyvolume for a while and greatly appreciating it, but there’s one thing I can’t figure out how to do: how the heck do I add a title to a figure/plot? I've tried various things, poked around on the Web... I can't figure it out.

Context: as installed by pip 22.1.2 (Python 3.9.5), I'm running the latest version of IPyvolume, 0.5.2, on the latest version of the Jupyter client, 7.1.2. An example plot I’m working with is created by ipv.scatter.

Thanks in advance!

**Comment by maartenbreddels** [comment, marked as answer]
Hi,

why does the label need to be in the Figure?
you could add a label before the figure like this:
```
import ipywidgets as widgets
display(widgets.Label(value="My label"))
```
Does that make sense?
